src/genie/libs/parser/iosxe/tests/ShowApConfigGeneral/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,115 @@
+expected_output = {
+    "ap_config_general_info": {
+        "bg-1-cap1": {
+            "cisco_ap_identifier": "70b3.d278.e03e",
+            "country_code": "IN",
+            "regulatory_domain_allowed_by_country": "802.11bg:-A   802.11a:-DN",
+            "ap_country_code": "IN  - India",
+            "slot_0": "-A",
+            "slot_1": "-D",
+            "mac_address": "70b3.1711.acbb",
+            "ip_address_configuration": "DHCP",
+            "ip_address": "10.10.5.14",
+            "ip_netmask": "255.255.254.0",
+            "gateway_ip_address": "10.10.5.1",
+            "capwap_path_mtu": 1485,
+            "capwap_active_window_size": 1,
+            "telnet_state": "Disabled",
+            "cpu_type": "ARMv7 Processor rev 1 (v7l)",
+            "memory_type": "DDR3",
+            "memory_size": "1028096 KB",
```

Comment:
It seems the unit is always `KB`. let's have the unit "KB" in key name and value is just number.
```
"memory_size_kb": 1028096,

Reply:
Updated